.gitignore

```diff
@@ -10,3 +10,5 @@ dist/
 # Testing-related
 .hypothesis/
 
+# Documentation
+site/
```

Comment:
This is the default location used by mkdocs,  I assume? I don't see `site/` referenced anywhere else in these changes...

Reply:
Yes, this is a default location when you are building the site locally.
The site is not referenced anywhere, because the GitHUb action for building and deploying the docs is building the documentation in a separate branch: https://github.com/tweag/FawltyDeps/tree/gh-pages, where only documentation sits.